pkg/scheduler/metrics/metrics.go

```diff
@@ -78,7 +78,7 @@ var (
 			Name:      "preemption_victims",
 			Help:      "Number of selected preemption victims",
 			// we think #victims>50 is pretty rare, therefore [50, +Inf) is considered a single bucket.
```

Comment:
The last bucket is now 64+, update the comment.

Reply:
Done